### DISCUSSION TITLE
Links Data Source

### DISCUSSION THREAD

**Discussion by ck-australia** [thurban/plugin_weathermap#114] [Q&A]
When I go to add a link between two devices and I add the device name to get the interface Im just getting the attached and cant tell which interface it is I want, when I do graph selector its showing the names, any ideas?

![image](https://user-images.githubusercontent.com/77236712/229510578-6183b795-9c71-4851-8b15-0fe0d088d148.png)

![image](https://user-images.githubusercontent.com/77236712/229510756-d33e54c7-200c-44d4-93ca-a2c0365e8417.png)

**Comment by vKnmnn** [comment]
Your Datasources do not contain the information in their name.

What i do is set the Name of the Data Source Template for traffic to "|host_description| - Traffic - |query_ifName| |query_ifAlias|"
then reindex the hosts and run reapply data source names script. Then the data source will contain more info.
